docs: Add release notes for 1.5.5 (#1924)
scalacenter/bloop: notes/v1.5.5.md

# bloop `v1.5.5`

> Bloop v1.5.5 is a bugfix release.

Reviewer: I'd also add something like the following

> One notable change this release for anyone using the `bloop-config` artifact is that it's not longer contained inside of this repo or versioned along with it. You can find its new location in [`scalacenter/bloop-config`](https://github.com/scalacenter/bloop-config). The latest version of `bloop-config` is still 1.5.5 and you'll continue to use it as your normally have.

Author: Added! Good idea!

> Bloop v1.5.5 is a bugfix release.

Reviewer 2: Bloop v1.5.5 is a bugfix release, but it also contains a major version bump of the `scala-debug-adapter` with [many new features](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.1).

So I would appreciate some advertisement of that.

> Bloop v1.5.5 ships a new major version of the Scala Debug Adapter, featuring better debugging steps (step filter), conditional breakpoint, logpoints and more. See a full lists of features in the releases notes of the Scala Debug Adapter [3.0.1](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.1) and [3.0.3](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.3).

Reviewer 3: If the classpath of projects depending on bloop artifacts now contains the new version of the debugger, it would be better to also bump the major version of bloop (since the new classpath is not binary compatible with the old classpath).

Reviewer 2: Bloop frontend is a tool not a library, so I don't think we need to follow the semantic versioning rules. Scala Debug Adapter uses `VersionScheme.Strict` which means that we can break the binary compatibility between any two versions, even a patch version. As a consequence, on Bloop, it can also break the binary compatibility between any two patch versions.

Author: Let's do that in a follow up then.

>So I would appreciate some advertisement of that. Something like the following:

Added! Though I think it will be better visible within Metals, which I also plan to add.

# bloop `v1.5.5`

> One of the notable changes in this release for anyone using the bloop-config

Reviewer: ```suggestion
One of the notable changes in this release for anyone using the `bloop-config`
```

artifact is that it's no longer contained inside of this repo or versioned along
with it. You can find its new location in
[scalacenter/bloop-config](https://github.com/scalacenter/bloop-config). The

> latest version of bloop-config is still 1.5.5 and you'll continue to use it as

Reviewer: ```suggestion
latest version of `bloop-config` is still 1.5.5 and you'll continue to use it as
```

> your normally have.

Reviewer: ```suggestion
you normally have.
```

Bloop v1.5.5 also ships a new major version of the Scala Debug Adapter,

> featuring better debugging steps (step filter), conditional breakpoint,
> logpoints and more. See a full lists of features in the release notes of the

Reviewer 2: ```suggestion
featuring better debugging steps (step filter), conditional breakpoints,
logpoints and more. See a full list of features in the release notes of the
```
